datajunction-server/datajunction_server/sql/parsing/ast.py

```diff
@@ -684,7 +684,7 @@ def type(self):
             self.add_type(self.expression.type)
             return self.expression.type
 
-        raise DJParseException(f"Cannot resolve type of column {self}.")
+        raise DJParseException(f"Cannot resolve type of column {self} in {self.parent}")
```

Comment:
What about something like:
```py
parent = f"in {self.parent}" if self.parent else "that has no parent"
raise DJParseException(f"Cannot resolve type of column {self} {parent}")
```
Then the example you updated in the test would be "Cannot resolve type of column status that has no parent".

Reply:
Good point, just updated!